server/routerlicious/packages/services-core/src/tenant.ts

```diff
@@ -17,6 +17,12 @@ export interface ITenantConfig {
 
 	customData: ITenantCustomData;
 
+	// Indicates if keyless access is enabled for this tenant.
+	enableKeylessAccess: boolean;
```

Comment:
nit: I find the `keyless` naming confusing, because they actual design/implementation _does_ use a Key, it's just private.

I don't feel super strongly about this, but I think one of the 2 below options would be good:
1. Rename `keyAccess` and `keylessAccess` to `sharedKeyAccess` and `privateKeyAccess` respectively. I think this naming more accurately represents what's going on.
2. Add a note to the doc-comment here explaining that "keyless access" actually means "use a private key to generate and sign tokens within the service"